templates/api-template-nodejs/components/logger.js

```diff
@@ -89,16 +89,14 @@ module.exports = function () {
 
     const log = (level, message) => {
         const timestamp = new Date().toISOString();
```

Comment:
remove timestamp variable as it is no longer used

Reply:
another format updated.